tracer/test/test-applications/integrations/Samples.AWS.SimpleNotificationService/Program.cs

```diff
@@ -2,22 +2,19 @@
 using System.Threading.Tasks;
 using Amazon.Runtime;
 using Amazon.SimpleNotificationService;
-using Amazon.SimpleNotificationService.Model;
 
-namespace Samples.AWS.SNS
+namespace Samples.AWS.SimpleNotificationService
 {
     public class Program
     {
         private static async Task Main(string[] args)
         {
             var snsClient = GetAmazonSimpleNotificationServiceClient();
 
-            string topicName = "MyTopic";
-            string topicArn = await GetOrCreateTopicAsync(snsClient, topicName);
-            string message = "Hello, SNS!";
-
-            await PublishMessageAsync(snsClient, topicArn, message);
-            await DeleteTopicAsync(snsClient, topicArn);
+#if NETFRAMEWORK
```

Comment:
Correct, we don't have any in the tracer, since we don't have any integration for receiving (an incoming trace). As far as I remember, all of this is being done (for Serverless) in the Datadog Extension, which has extensive unit testing – and I imagine integration testing, for continuing traces through these type of AWS SDK integrations!

Reply:
I believe that there are now Kafka integration tests being added to the system-tests repo, with other messaging systems planned to be added. Let's keep an eye out for further updates there, but that will be the best way to ensure context propagation works correctly both within our tracer and across tracers

Reply:
Ah I see, the Serverless team does not participate in the `system-tests` repo as of now. Since our product would require a more complex setup for it, yet I advocate to join the effort in the future! Just so we ensure context propagation works properly across tracers and the Serverless product!